src/transformers/models/oneformer/processing_oneformer.py

```diff
@@ -37,34 +70,43 @@ class OneFormerProcessor(ProcessorMixin):
             The image processor is a required input.
         tokenizer ([`CLIPTokenizer`, `CLIPTokenizerFast`]):
             The tokenizer is a required input.
-        max_seq_len (`int`, *optional*, defaults to 77)):
+        max_seq_length (`int`, *optional*, defaults to 77):
             Sequence length for input text list.
-        task_seq_len (`int`, *optional*, defaults to 77):
+        task_seq_length (`int`, *optional*, defaults to 77):
             Sequence length for input task token.
     """
 
     attributes = ["image_processor", "tokenizer"]
     image_processor_class = "OneFormerImageProcessor"
     tokenizer_class = ("CLIPTokenizer", "CLIPTokenizerFast")
+    optional_call_args = ["task_inputs", "segmentation_maps"]
 
     def __init__(
-        self, image_processor=None, tokenizer=None, max_seq_length: int = 77, task_seq_length: int = 77, **kwargs
+        self,
+        image_processor=None,
+        tokenizer=None,
     ):
```

Comment:
Sorry I hadn't caught that before, but removing max_seq_length and task_seq_length from the init looks like a breaking change. We should add them back, and use self.max_seq_length and self.task_seq_length in the processing when the "max_length" kwarg is not defined. No need to add `max_seq_length` and `task_seq_length` to `OneFormerTextKwargs` as they weren't accepted before.